Microsoft.DotNet.Interactive.PowerShell/Host/PSKernelHost.cs

```diff
@@ -71,18 +71,18 @@ public override void SetShouldExit(int exitCode)
 
         #region "IHostSupportsInteractiveSession Implementation"
 
-        public bool IsRunspacePushed => throw new NotImplementedException("IHostSupportsInteractiveSession support is coming soon.");
+        public bool IsRunspacePushed => false;
 
-        public Runspace Runspace => throw new NotImplementedException("IHostSupportsInteractiveSession support is coming soon.");
+        public Runspace Runspace => throw new PSNotImplementedException("IHostSupportsInteractiveSession support is coming soon.");
```

Comment:
It might be helpful to include a link to an issue in the error.

Reply:
I'm evaluating the feasibility to support this interface and so far it doesn't look good (though I'm not super sure yet) as we don't really use the host to read code input and initiate the code execution. For the ps kernel, it's mainly used to display results. So I might remove the support of this interface once I'm sure about this.